### PR TITLE
Checker

### PR DESCRIPTION
**Pull request by baminou**

### CODE REVIEW

**Comment by junjun-zhang** [comment]
run the checker successfully with: `cwltool --debug checker-workflow-wrapping-workflow.cwl checker-input-cwl.json`

**Review by junjun-zhang — APPROVED** [review]
good to merge

**Comment by agduncan94** [comment]
Looks good, I tested running on Dockstore through the CLI. https://dockstore.org/workflows/github.com/ICGC-TCGA-PanCancer/Seqware-BWA-Workflow/_cwl_checker:checker?tab=info

File: checker-workflow-wrapping-workflow.cwl [hunk 1]
```diff
@@ -0,0 +1,134 @@
+cwlVersion: v1.0
+class: Workflow
+
```
Comment (denis-yuen) [inline checker-workflow-wrapping-workflow.cwl:3]: Doesn't have to be done immediately, but a small `doc:` section would be helpful 
i.e. this is a workflow that checks on the output of bwa by running md5sum on the results

Example at
http://www.commonwl.org/user_guide/17-metadata/

**Review by denis-yuen — APPROVED** [review]

File: checker-workflow-wrapping-workflow.cwl (hunk 1 above)
Comment (junjun-zhang) [reply checker-workflow-wrapping-workflow.cwl:3]: Thanks @denis-yuen, @baminou is off today, we will do that later.

Comment (junjun-zhang) [reply checker-workflow-wrapping-workflow.cwl:3]: Added an issue for doc: #17

**Pull request merged**